README.md

```diff
@@ -81,6 +81,92 @@ output:
 
 It is not possible to mix two syntaxes through a single render call.
 
+## User-Defined Syntax
+
+Alternatively, you can specify your own templating delimiter syntax.
+This is useful to prevent template collision, in similar cases to semblance,
+but is more flexible.
+
+Input markup (`test.html` file):
+
+    <h1><%= title %></h1>
+    <% each items, item %>
+        <h2><%= item.title %></h2>
+        <div class="content"><%~ item.content %></div>
+    <% end %>
+
+rendering with data:
+
+    use_module(library(st/st_render)).
+
+    current_output(Out),
+    st_render_file(test, _{
+        title: 'Hello',
+        items: [
+            _{ title: 'Item 1', content: 'Abc 1' },
+            _{ title: 'Item 1', content: 'Abc 2' }
+        ]
+    }, Out, _{
+        frontend: syntax_tokens(
+            comment("<%#", "%>"),
+            out("<%=", "%>"),
+            out_unescaped("<%~", "%>"),
+            statement("<%", "%>")
+        )
```

Comment:
The `frontend` option can still be passed `simple` or `semblance`, but can also be passed this underlying `syntax_tokens` data that defines delimiters for templating. The system is fully compatible, no breaking changes.